src/settings.cc

```diff
@@ -89,9 +89,9 @@ Settings::~Settings()
 
 void Settings::visit(SettingsVisitor& visitor)
 {
-	for (std::list<SettingsEntry *>::iterator it = entries.begin();it != entries.end();it++) {
```

Comment:
indentation mismatch

Reply:
fixed - the file was using a mix of four space and tab indentation